torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1588,7 +1588,16 @@ that adds a prefix to each key inserted to the store.
     prefix (str): The prefix string that is prepended to each key before being inserted into the store.
     store (torch.distributed.store): A store object that forms the underlying key-value store.
       )")
-      .def(py::init<const std::string&, c10::intrusive_ptr<::c10d::Store>>())
```

Comment:
The previous behavior looks very suspicious that makes me even wonder if there is a specific purpose. (lol)

Reply:
im not sure either hahaha, lets see from the tests. Though there were very few PrefixStore tests from what i can tell